R/mng01.R

```diff
@@ -103,20 +106,37 @@ mng01_main <- function(adam_db,
     subject_var = "USUBJID"
   )
 
-  if (!is.null(names(line_col))) {
-    color_lvl <- sort(unique(df[[arm_var]]))
-    col <- line_col[as.character(color_lvl)]
+
+  arm_lvl <- sort(unique(df[[arm_var]]))
+  col <- if (!is.null(names(line_col))) {
+    col <- line_col[as.character(arm_lvl)]
```

Comment:
I know it works anyway but to avoid confusion can you use a different name for the internal var?

Reply:
good point, let me correct that.